<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Download Image</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEditURL">
     <property name="placeholderText">
      <string>Enter image URL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDownload">
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelImage"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

